tests/shouldwork/Cores/Xilinx/Floating/Annotations.hs

```diff
@@ -32,3 +33,26 @@ binEnTopAnn name =
           ]
     , t_output = PortName "result"
     }
+
+unTopAnn :: String -> TopEntity
+unTopAnn name =
```

Comment:
I suppose the `un` is for `unary`? This took me 30 seconds to figure out. Admittedly, I'm still somewhat covid-brained but.. :-). Could you spell it out?

Reply:
Do you also want me to spell out "binary" in "binTopAnn" then? I admit the name is pretty cryptic, I actually had a laugh about it, and I wouldn't put it in code exposed to our users. You picked the benign one, "unEnTopAnn" is almost like an exercise for your vocal cords.

Reply:
For symmetry's sake let's :)